.buildkite/scripts/steps/functional/security_cypress_exec.sh

```diff
@@ -5,9 +5,12 @@ set -euo pipefail
 source .buildkite/scripts/steps/functional/common.sh
 source .buildkite/scripts/steps/functional/common_cypress.sh
 
-export JOB=kibana-threat-intelligence-chrome
+export JOB=$JOB_TITLE
 export KIBANA_INSTALL_DIR=${KIBANA_BUILD_LOCATION}
 
-echo "--- Threat Intelligence Cypress tests (Chrome)"
+echo $MSG
 
-yarn --cwd x-pack/plugins/threat_intelligence cypress:run
+cd $CURRENT_WORKING_DIR
+
+set +e
+yarn $TARGET; status=$?; yarn junit:merge || :; exit $status
```

Comment:
Thanks for doing this, it looks great!

Can we verify $TARGET is one of the available options listed in on_merge.yml and base.yml?  It's going to be injected from an external service and it would be safer to validate it's an expected command.

Reply:
I think this addition handles this check. https://github.com/elastic/kibana/pull/172151/commits/f5a0bbfb8a3152ac61817c6a6fc9b398a0f661e7